pilot/docker/Dockerfile.proxyv2

```diff
@@ -15,6 +23,12 @@ RUN chown -R istio-proxy /var/lib/istio
 
 COPY istio-iptables /usr/local/bin/istio-iptables
 
+# Add CAP_NET_ADMIN and CAP_NET_RAW as Permitted capabilities
+# to the istio-iptables binary.
+COPY --from=libpcapbuilder /sbin/setcap /sbin
+RUN /sbin/setcap cap_net_admin,cap_net_raw+p /usr/local/bin/istio-iptables \
```

Comment:
Can we just do this in the makefile (eg on the host machine, not in docker) before we copy it in to avoid all the docker complexity of instlaling, removing, etc

Reply:
I doubt that COPY preserves extended attributes.

Reply:
Can you try it? We already do this with chmod, see BUILD_PRE (I think)

Reply:
Doesn't work. `COPY` doesn't preserve extended attributes:
```
mkdir /tmp/testcap
cd /tmp/testcap

touch test.txt
sudo /sbin/setcap cap_net_admin,cap_net_raw+p test.txt


cat >Dockerfile.captest <<EOF
FROM ubuntu:bionic

RUN apt-get update \
&& apt-get upgrade -y \
&& apt-get install --no-install-recommends -y \
    libcap2-bin \
&& apt-get clean \
&& rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old

COPY test.txt /

ENTRYPOINT [ "/sbin/capsh" ]
CMD [ "--print" ]
EOF

docker build -f Dockerfile.captest -t istio/testcap:latest .

docker run --rm --name=captest --entrypoint "" istio/testcap:latest /sbin/getcap /test.txt
```
Outputs nothing, e.g. the copied `/test.txt` file has no capabilities.

Reply:
:frowning_face: